lib/core/src/Cardano/Wallet/CoinSelection/Internal/Balance.hs

```diff
@@ -849,8 +851,12 @@ performSelectionEmpty performSelectionFn constraints params =
     transform :: a -> (NonEmpty (Address ctx, TokenBundle) -> a) -> a
     transform x y = maybe x y $ NE.nonEmpty $ view #outputsToCover params
 
+
+    -- FIXME: How does the choice of address matter here?
+    dummyAddress = longestChangeAddress constraints
```

Comment:
If only a dummy address matters here, I think it should be named `Balance.SelectionConstraints.dummyAddress` instead of `Balance.SelectionConstraints.longestChangeAddress`

Reply:
But it seems we do need `longestChangeAddress` here, because we use it in
```haskell
            { minCoinFor = computeMinimumAdaQuantity longestChangeAddress
```

Reply:
Dropped the `FIXME`.